src/MongoDB.Driver.Core/Core/WireProtocol/CommandUsingCommandMessageWireProtocol.cs

```diff
@@ -328,12 +330,6 @@ private void MessageWasProbablySent(CommandRequestMessage message)
             {
                 _session.WasUsed();
             }
-
-            var transaction = _session.CurrentTransaction;
-            if (transaction != null && transaction.State == CoreTransactionState.Starting)
-            {
-                transaction.SetState(CoreTransactionState.InProgress);
-            }
```

Comment:
Leave this lines in. No changes required to the `MessageWasProbablySent` method.